Library/Formula/libhdhomerun.rb

```diff
@@ -0,0 +1,14 @@
+require "formula"
+
+class Libhdhomerun < Formula
+  homepage "https://www.silicondust.com/support/downloads/linux/"
+  url "http://download.silicondust.com/hdhomerun/libhdhomerun_20140604.tgz"
+  sha1 "b8d910a5721c484a30b81662fd6991e3546ed67c"
+
+  def install
+    system "make"
+    bin.install "hdhomerun_config"
+    lib.install "libhdhomerun.dylib"
+    include.install Dir["hdhomerun*.h"]
+  end
+end
```

Comment:
Or just run with `shell_output` and provide the expected return code for e.g. `--version` or `--help`.